src/dragonball/src/dbs_virtio_devices/src/block/request.rs

```diff
@@ -83,7 +83,6 @@ impl RequestHeader {
     /// need to do an explicit little endian read as all reads are little endian by default.
     /// When running on a big endian platform, this code should not compile, and support
     /// for explicit little endian reads is required.
-    #[cfg(target_endian = "little")]
```

Comment:
The comment clearly explains that this is a virtio 1.0 device model and little endian reads are required.